css/style.css

```diff
@@ -381,15 +390,127 @@ span::before {
 }
 
 .form-button:hover {
-  background-color: #dfe1e6;
-  color: black;
+  background-color: #6070ff;
+  color: #fff;
+  border: solid 2px #fff;
 }
 
 .btn:hover {
   background-color: #6070ff;
   color: #fff;
 }
 
+/* Work Modal Styles */
+.workModal {
```

Comment:
- [ ] You have done a great job styling your modal popup for desktop view. It works perfectly. However, note that according to the project requirements you are supposed to make the modal popup work both responsively and function as required for the mobile version. Currently, when I tap the modal popup for the mobile version, I cannot be able to scroll until I either reload or switch my screen orientation back to desktop view. KIndly fix this.

Reply:
i decided to restyle everything from scratch, it was a good practice and it helped me to identify where i went wrong

Reply:
I believe this is even better :clap: